throw new Error("oops");
console.log("after");
